xds/internal/xdsclient/xdsresource/unmarshal_cds.go

```diff
@@ -503,7 +503,7 @@ func outlierConfigFromCluster(cluster *v3clusterpb.Cluster) (*OutlierDetection,
 	interval := defaultInterval
 	if i := od.GetInterval(); i != nil {
 		if err := i.CheckValid(); err != nil {
-			return nil, fmt.Errorf("outlier_detection.interval is invalid with error %v", err)
+			return nil, fmt.Errorf("outlier_detection.interval is invalid with error: %v", err)
```

Comment:
Can we skip the `with error` part in these error strings:
`return nil, fmt.Errorf("outlier_detection.interval is invalid: %v", err)`